css/home-page.styl

```diff
@@ -439,6 +440,10 @@ $flex-section
           &:nth-child(odd)
             justify-content: flex-end
 
+        @media (max-width: 400px)
+          flex-basis: 100%
+          flex-direction: column
+
```

Comment:
This extended media query was necessary as `flex-basis` is used primarily in the above media queries to keep project cards breaking two at a time instead of stranding a sole card on a line alone. Mobile Safari wasn't breaking into column with the small viewport.